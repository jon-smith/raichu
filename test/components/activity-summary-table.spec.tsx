import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ActivitySummaryTable, { ActivityData } from '@renderer/components/activity-summary-table';

const testData: ActivityData[] = [
	{ filename: '1', name: 'one', date: new Date(2000, 1, 1) },
	{ filename: '2', name: 'two' }
];

describe('Counter component', () => {
	it('renders correctly', () => {
		const tree = renderer.create(<ActivitySummaryTable rows={testData} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
