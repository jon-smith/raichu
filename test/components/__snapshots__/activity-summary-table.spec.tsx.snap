// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <table
      aria-label="activity table"
      className="MuiTable-root makeStyles-table-3"
    >
      <thead
        className="MuiTableHead-root"
      >
        <tr
          className="MuiTableRow-root MuiTableRow-head"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeSmall"
            scope="col"
          >
            Filename
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeSmall"
            scope="col"
          >
            Name
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight MuiTableCell-sizeSmall"
            scope="col"
          >
            Date
          </th>
        </tr>
      </thead>
      <tbody
        className="MuiTableBody-root"
      >
        <tr
          className="MuiTableRow-root"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeSmall"
            scope="row"
          >
            1
          </th>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            one
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            Tue Feb 01 2000 00:00:00 GMT+0000 (Greenwich Mean Time)
          </td>
        </tr>
        <tr
          className="MuiTableRow-root"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeSmall"
            scope="row"
          >
            2
          </th>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            two
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeSmall"
          >
            undefined
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
